Assets/_Project/Runtime/_Scripts/UI Scripts/ScoreManager.cs

```diff
@@ -0,0 +1,62 @@
+using TMPro;
+using UnityEngine;
+
+public class ScoreManager : MonoBehaviour
+{
+    [Header("Current UI Elements")]
+    [SerializeField] TextMeshProUGUI currentDepthScore;
+    [SerializeField] TextMeshProUGUI currentAlgaeCollected;
+    [SerializeField] TextMeshProUGUI currentHolesRepaired;
+    [SerializeField] TextMeshProUGUI currentJellyfishCollected;
+    [SerializeField] TextMeshProUGUI currentLifeformsScanned;
+
+    [Header("Current Scores")]
+    public int depthScore;
+    public int algaeCollected;
+    public int holesRepaired;
+    public int jellyfishCollected;
+    public int lifeformsScanned;
```

Comment:
Don't make public variables. Use properties instead.
Example:

```c#
[SerializeField] int depthScore;

public int DepthScore => depthScore; // get only property

// or:

public int DepthScore
{
    get => depthScore;
    set => depthScore = value;
}
```

Reply:
Fixed it! Did the same thing with high score